src/styles/blocks/header.scss

```
@@ -0,0 +1,82 @@
+body {
+  margin: 0;
+}
+
+header {
```

use class selector instead of tag one
```suggestion
.header {
```